src/cmds/openapi.ts

```diff
@@ -227,12 +228,11 @@ export default class OpenAPICommand extends Command {
       Command.debug(`api settings list response payload: ${JSON.stringify(apiSettingsBody)}`);
       if (!apiSettingsBody.length) return createSpec();
 
-      const { option }: { option: 'create' | 'update' } = await prompt(
+      const { option }: Answers<string> = await promptTerminal(
```

Comment:
I know we can't have `{ option: 'create' | 'update' }` typing anymore but the `Answers<string>` typing here is a little weird because it doesn't seem to be doing anything. Maybe remove it until we figure out how to better work with this `Answers<>` generic and our destructured answers?

![Screen Shot 2022-08-18 at 6 12 49 PM](https://user-images.githubusercontent.com/33762/185521163-ed8e7f50-4c30-4806-b289-a124a31405ab.png)

```suggestion
      const { option } = await promptTerminal(
```

Reply:
will do! i'm with you, it's not super helpful in its current form. i was kinda struggling with how to mangle with this generic and make it... less generic lol.

Reply:
i'm gonna rip them out from everywhere!